.github/workflows/publish-to-pypi.yml

```diff
@@ -35,13 +36,13 @@ jobs:
     - name: Install dependencies
       run: python -m pip install setuptools wheel
 
-    - name: Fix up version string
+    # This step is only necessary for testing purposes and for TestPyPI
+    - name: Fix up version string for TestPyPI
+      if: ${{ !startsWith(github.ref, 'refs/tags') }}
```

Comment:
If I understand it correctly, when we make a release/tag, this sed command won't run. So, we are pushing the same versions (e.g., `v0.3.1`) to both PyPI and TestPyPI, right?

Reply:
Yes, correct. But I think pushing v0.3.1 to both PyPI and TestPyPI is fine right?

Reply:
Yes, it looks good.